### DISCUSSION TITLE
Ferrari F355 crashes

### DISCUSSION THREAD

**Discussion by Juissi71** [flyinghead/flycast#1594] [General]
Hi. Naomi Ferrari F355 crashes after 10...20min in demo (attraction)mode. I haven't  played F355 much yet, so I don't know if it crashes when playing the game. I have tried OpenGL and DirectX9 and same result. Setup is Win7 64bit.

**Comment by Juissi71** [comment]
I tried yesterday again and it took about 5min to freeze F355 (twin/deluxe) in demo mode.
It happens in random part of the game.
Alt + F4 exits the game but it takes a longer time than exit running game.
Texture memory is set to max. 1024.

Last lines of the log-file are: 
38:40:981 rend/dx9/dxcontext.cpp:120 N[RENDERER]: DX9 Context terminating
40:12:665 hw/naomi/multiboard.cpp:183 E[NAOMI]: Time out waiting for multiboard vsync. Slave 0
40:17:666 hw/naomi/multiboard.cpp:183 E[NAOMI]: Time out waiting for multiboard vsync. Slave 0
40:17:708 rend/dx9/dxcontext.cpp:120 N[RENDERER]: DX9 Context terminating

**Comment by flyinghead** [comment]
Crashes happen randomly with this game. This is due to the tight synchronization between instances. Reasonably fast and recent CPU and GPU are needed. Disabling VSync (including in external utilities like RivaTuner) is highly recommended. Using only one screen (twin) works better than three screens (deluxe). Different graphics API (dx11, vulkan, etc.) can also give different results.

Once a desync occurs, quitting any instance can be slow due to the long timeout of sync points.

**Reply by Juissi71** [reply]
Thank you for answer. My arcade cab's PC runs Naomi games 60fps but I think, it's a little slow. I5 3nd gen.
When I disabled VSync game runs longer, about 30mins. My desktop is faster and it works well.
